app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/CompleteClaim.scala

```diff
@@ -68,7 +68,7 @@ object CompleteClaim {
     maybeCompleteBankAccountDetailAnswer: Option[CompleteBankAccountDetailAnswer],
     supportingEvidenceAnswers: CompleteSupportingEvidenceAnswer,
     completeCommodityDetailsAnswer: CompleteCommodityDetailsAnswer,
-    completeNorthernIrelandAnswer: CompleteNorthernIrelandAnswer,
+    completeNorthernIrelandAnswer: Option[CompleteNorthernIrelandAnswer],
```

Comment:
Isn't this a mandatory question?

Reply:
Yes, but only if the NI feature flag is enabled.  If the feature flag is disabled then there isn't an NI answer, so there isn't anything for completeClaim to process.

Reply:
This breaks the semantics of the current model. We will need to discuss this.